manifests/g/GadgetsRevived/GadgetsRevived/1.0/GadgetsRevived.GadgetsRevived.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: GadgetsRevived.GadgetsRevived
+PackageVersion: 1.0
+PackageLocale: en-US
+Publisher: GadgetsRevived
+PackageName: Gadgets Revivied

```

```suggestion
PackageName: Windows Desktop Gadgets
```
Should Match Control Panel